dotcom-rendering/src/lib/assets.ts

```diff
@@ -15,16 +15,24 @@ try {
 	// do nothing
 }
 
-// TODO: this should be removed in favour of `frontendAssetsFullURL` defined in CAPI
-// GU_STAGE is set in cloudformation.yml, so will be undefined locally
-const stage =
-	typeof process.env.GU_STAGE === 'string'
-		? process.env.GU_STAGE.toUpperCase()
```

Comment:
Do we always receive this env var in capitals or is it worth including `toUpperCase()` in the new flow?

Reply:
That's actually a really good point!

🤔 I don't believe it will be different but if that transform was there before it means someone thought that it could so we should respect that. I've updated the code for this.